src/api/module/history/entity/index.ts

```diff
@@ -1,27 +1,29 @@
+import { Option } from '@/api/shared/entity';
+
 export type History = {
   id?: number;
-  murojaahType: number;
+  historyType: number;
   juz?: number;
   surah?: number;
   surahName?: string;
   start?: number;
   end?: number;
-  murojaahMethod: string;
-  totalMurojaah: number;
+  approachId: number;
+  repeat: number;
   occuredAt: Date;
 };
 
-export enum MurojaahType {
+export type Payload = {
```

Comment:
maybe a more meaningful name than `Payload`?

Reply:
currently will use this, because if need to be specific then it should be already specific because it's in module history, if need other name than payload then we can use alias like HIstoryPayload when importing history's payload, this same like other naming in service and repository too

later if I think this is still confusing maybe will refactor it, thank you